### PR TITLE
Update FAQ entries about recovery vaccination status

### PR DESCRIPTION
**Pull request by larswmh**
As a response to [this issue](https://github.com/corona-warn-app/cwa-website/issues/1713), here's the pull request to include the text mentioned in the issue to the mentioned FAQ articles.

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
                         "anchor": "rec_cert",
                         "active": true,
                         "textblock": [
-                            "Ab der Version 2.5 der Corona-Warn-App werden Genesenenzertifikate unterstützt."
+                            "Ab der Version 2.5 der Corona-Warn-App werden Genesenenzertifikate unterstützt. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeitentfällt bei allen Impfstoffen außer bei Johnson & Johnson."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:110]:
```suggestion
                            "Ab der Version 2.5 der Corona-Warn-App werden Genesenenzertifikate unterstützt. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson & Johnson."
```

File: src/data/faq_de.json [hunk 2]
```diff
@@ -147,7 +147,7 @@
...
-                            "Bei einem​ Genesenenzertifikat ist die fachliche Gültigkeit laut des EU eHealth Networks festgelegt auf 'Datum des ersten positiven PCR-Test-Ergebnisses der Person' plus 180 Tage.​",
+                            "Bei einem​ Genesenenzertifikat ist die fachliche Gültigkeit laut des EU eHealth Networks festgelegt auf 'Datum des ersten positiven PCR-Test-Ergebnisses der Person' plus 180 Tage.​ Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeitentfällt bei allen Impfstoffen außer bei Johnson & Johnson.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:150]:
```suggestion
                            "Bei einem​ Genesenenzertifikat ist die fachliche Gültigkeit laut des EU eHealth Networks festgelegt auf 'Datum des ersten positiven PCR-Test-Ergebnisses der Person' plus 180 Tage.​ Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson & Johnson.",
```

File: src/data/faq_de.json [hunk 3]
```diff
@@ -283,7 +283,7 @@
...
+                            "Ab Version 2.3 der Corona-Warn-App (CWA) können Sie Ihren digitalen Impfnachweis in der CWA hinzufügen und so per QR-Code Ihren Corona-Impfstatus bei Bedarf sicher und digital nachweisen. Ab Version 2.5 der App können digitale Impfnachweise für mehrere Personen in der App gespeichert werden. Die CWA zeigt den vollständigen Impfschutz 14 Tage nach der letzten benötigten Impfung an. Der digitale Impfnachweis ist eine zusätzliche Möglichkeit, um Impfungen zu dokumentieren. Sie können damit Informationen wie Impfzeitpunkt und Impfstoff personalisiert bequem auf Ihren Smartphones digital speichern. Die Nutzung des digitalen Impfnachweises in der CWA ist freiwillig. Sie können eine Corona-Impfung auch auf eine andere Weise nachweisen. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeitentfällt bei allen Impfstoffen außer bei Johnson & Johnson."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:286]:
```suggestion
                            "Ab Version 2.3 der Corona-Warn-App (CWA) können Sie Ihren digitalen Impfnachweis in der CWA hinzufügen und so per QR-Code Ihren Corona-Impfstatus bei Bedarf sicher und digital nachweisen. Ab Version 2.5 der App können digitale Impfnachweise für mehrere Personen in der App gespeichert werden. Die CWA zeigt den vollständigen Impfschutz 14 Tage nach der letzten benötigten Impfung an. Der digitale Impfnachweis ist eine zusätzliche Möglichkeit, um Impfungen zu dokumentieren. Sie können damit Informationen wie Impfzeitpunkt und Impfstoff personalisiert bequem auf Ihren Smartphones digital speichern. Die Nutzung des digitalen Impfnachweises in der CWA ist freiwillig. Sie können eine Corona-Impfung auch auf eine andere Weise nachweisen. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson & Johnson."
```

File: src/data/faq_de.json [hunk 4]
```diff
@@ -312,7 +312,7 @@
...
                             "Das digitale Impfzertifikat der EU kann als ausgedruckter QR-Code oder als Scan in der Corona-Warn-App oder CovPass-App vorgezeigt werden. Der bekannte Impfnachweis in Form des gelben Impfausweises bleibt weiterhin gültig. Das neue digitale Impfzertifikat der EU für eine COVID-19-Impfung stellt lediglich eine nutzerfreundliche und vertrauenswürdige Ergänzung zum gelben Impfausweis dar, die einfach geprüft werden kann.",
-                            "Bitte beachten Sie, dass eine ausreichende Wirksamkeit des Impfschutzes erst 14 Tage nach der letzten notwendigen Impfung eintritt.",
+                            "Bitte beachten Sie, dass eine ausreichende Wirksamkeit des Impfschutzes erst 14 Tage nach der letzten notwendigen Impfung eintritt. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeitentfällt bei allen Impfstoffen außer bei Johnson & Johnson.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:315]:
```suggestion
                            "Bitte beachten Sie, dass eine ausreichende Wirksamkeit des Impfschutzes erst 14 Tage nach der letzten notwendigen Impfung eintritt. Sofortige Impfstatusanzeige für Genesene: Für Genesene wird ab der Version 2.9 direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson & Johnson.",
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Missing spaces in "Wartezeitentfällt"

**Comment by larswmh** [comment]
> Missing spaces in "Wartezeitentfällt"

Added the spaces in my latest commit, thanks!

**Comment by dsarkar** [comment]
@MikeMcC399 @larswmh Thanks!

**Pull request merged**